src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -1110,10 +1090,7 @@ void AppHost::_ShowWindowChanged(const winrt::Windows::Foundation::IInspectable&
     // should prevent scenarios where the Terminal window state and PTY window
     // state get de-sync'd, and cause the window to minimize/restore constantly
     // in a loop.
-    if (_showHideWindowThrottler)
-    {
-        _showHideWindowThrottler->Run(args.ShowOrHide());
-    }
+    _showHideWindowThrottler->Run(args.ShowOrHide());

```

Comment:
This `if` check is a common pattern we got with our throttled funcs. It's generally not very safe, because it fails to work as soon as `Run()` is called from a background thread.